src/main/java/com/commercetools/sync/services/ShoppingListService.java

```diff
@@ -0,0 +1,117 @@
+package com.commercetools.sync.services;
```

Comment:
This service is not part of the other PR, interesting.

Reply:
yes Hin added this because he needs it in ReferenceResolver